examples/complex/esp32/ios-gateway/extra/iOS_IoTCore_Client.xcodeproj/project.xcworkspace/xcuserdata/danilo.xcuserdatad/IDEFindNavigatorScopes.plist

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<array/>
+</plist>
```

Comment:
I cannot comment on the binary personal data file you checked in (galzahavi.xcuserdata) but this and other files specific to your account should not be checked in.